content/applications/marketing/events/event_booths.rst

```diff
@@ -0,0 +1,243 @@
+proceed to make any necessary changes from the booth form. Or, create a new one, by clicking the
+:guilabel:`New` button in the upper-left corner to reveal a blank booth form.
+
+Booth form
+----------
+
+The booth form in Odoo *Events* lets users customize and configure event booths in a number of
+different ways.
+
+.. image:: event_booths/booth-form.png
+   :align: center
+   :alt: Typical booth form in the Odoo Events application.
+
+Start by typing in a :guilabel:`Name` for the booth. This is a **required** field.
+
+Then, apply a :guilabel:`Booth Category` to the booth. This is a **required** field.
+
+.. tip::
+   A new :guilabel:`Booth Category` can  be created from this field, by typing in the name of the
+   new category, and clicking :guilabel:`Create and edit...` from the resulting drop-down menu.
+   Doing so reveals a :guilabel:`Create Booth Category` pop-up window, with all the standard fields
+   found on a common booth category form.
+
+   Simply clicking :guilabel:`Create` from the resulting drop-down menu creates the category, but
+   does not reveal the :guilabel:`Create Booth Category` pop-up window. The category would have to
+   be modified later, via the *Booth Categories* page (:menuselection:`Events app --> Configuration
+   --> Booth Categories`).
+
+Upon selecting a pre-existing :guilabel:`Booth Category`, two additional, non-modifiable fields
+appear: :guilabel:`Product` and :guilabel:`Price`. Both fields represent their respective selections
+for that specific booth category.
+
+When a person purchases a booth rental through the event website, the subsequent renter-related
+fields on the form auto-populate, based on the information provided by the purchaser during the
+online transaction. The booth also automatically changes its status from *Available* to
+*Unavailable*.
+
+However, if the rental of a booth is conducted in any other way (e.g., in person, via sales order,
+etc.), the :guilabel:`Renter`, :guilabel:`Renter Name`, :guilabel:`Renter Email`, and
+:guilabel:`Renter Phone` fields can be entered in manually.
+
+The status of the booth can also be changed manually, either by clicking the appropriate status
+from the status bar present on the booth form, or by dragging-and-dropping the desired booth into
+the appropriate stage, via the *Booths* page Kanban view.
+
+Sell event booths
+=================
+
+With event booths configured in the back-end, via the event-specific *Booths* pages, Odoo presents
+them on the front-end of the event website, via the *Get A Booth* event subheader link.
+
+To access the *Get A Booth* page on the event website, open the :menuselection:`Events app`, and
+select the desired event from the :guilabel:`Events` dashboard. From the event form, click the
+:guilabel:`Go to Website` smart button to be taken to the front-end of the Odoo-built event website.
+
+From there, click the :guilabel:`Get A Booth` event subheader menu option. Doing so reveals the
```

Comment:
where is the "Get a Booth" event subheader menu? o;
![image](https://github.com/odoo/documentation/assets/23611711/7f05a81a-a7e5-45b6-a158-01d961c26018)

Also, are only "Unavailable" booths able to be booked?

Reply:
Woo hoo! I figured it out. And, GREAT CATCH on this! I've added a detailed admonition, with specific steps, for 2 different methods users can employ to ensure they see the 'Get A Booth' subheader menu option. It should definitely solve any issues they encounter. Thanks again for bringing this to my attention. 🙏